src/polymer-types.ts

```diff
@@ -32,5 +32,8 @@ declare global {
       success: boolean;
       response: unknown;
     };
+    haptic: {
+      hapticType: string;
```

Comment:
Register this event inside `util/haptics.ts`. Also for type, use the values of `ALLOWED_HAPTIC_TYPES`